### ISSUE TITLE
Natural Sort

### ISSUE THREAD

**Issue by jc211** [NVlabs/instant-ngp#247]
This should be a natural sort I think

https://github.com/NVlabs/instant-ngp/blob/26a85a22917bd7a3cad320c10873dd8acb4f3c0d/src/nerf_loader.cu#L219

**Comment by Tom94** [comment]
Yes, this is a good suggestion! We should use natural sorting here -- going to put it on the todo list.
